--- linux-4.11/drivers/Makefile~	2017-05-01 02:47:48.000000000 +0000
+++ linux-4.11/drivers/Makefile	2017-05-04 13:16:52.913221958 +0000
@@ -26,6 +26,9 @@
 # IPMI must come before ACPI in order to provide IPMI opregion support
 obj-y				+= char/ipmi/
 
+obj-$(CONFIG_VIRTIO)		+= virtio/
+obj-$(CONFIG_XEN)		+= xen/
+
 obj-$(CONFIG_ACPI)		+= acpi/
 obj-$(CONFIG_SFI)		+= sfi/
 # PnP must come after ACPI since it will eventually need to check if acpi
@@ -41,8 +44,6 @@
 # SOC specific infrastructure drivers.
 obj-y				+= soc/
 
-obj-$(CONFIG_VIRTIO)		+= virtio/
-obj-$(CONFIG_XEN)		+= xen/
 
 # regulators early, since some subsystems rely on them to initialize
 obj-$(CONFIG_REGULATOR)		+= regulator/
